components/alert/alert.css

```diff
@@ -0,0 +1,23 @@
+[class*="alert"] {
+  box-sizing: border-box;
+  border: none;
+  border-radius: 0.3rem;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.alert--success {
+  background: #d4edda;
```

Comment:
go for color variables, make a different file for colors and import here, i was thinking of using it that wat

Reply:
Good suggestion, thanks!